### PR TITLE
CVSB-995

### PR DESCRIPTION
**Pull request by vladddragomir**

### CODE REVIEW

File: src/pages/testing/test-creation/country-of-registration/country-of-registration.ts [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+@IonicPage()
+@Component({
+  selector: 'page-country-of-registration',
+  templateUrl: 'country-of-registration.html',
+})
+export class RegionReadingPage implements OnInit {
+  @ViewChild('searchBar') searchBar;
+  searchVal: string = '';
+  topElem = [];
```
Comment (danieliordache) [inline src/pages/testing/test-creation/country-of-registration/country-of-registration.ts]: add types

File: src/pages/testing/test-creation/country-of-registration/country-of-registration.html [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+                (click)="setVehicleRegCountry(elem)"
+                *ngFor="let elem of botElem">
+            <h3>{{ elem.value }}</h3>
+            <ion-icon item-end class="selected"
+                      name="checkmark"
+                      *ngIf="vehicle.countryOfRegistration === elem.key"></ion-icon>
+        </button>
+    </ion-list>
+
+    <div *ngIf="filteredCountries.length == 0" class="no-result">
```
Comment (danieliordache) [inline src/pages/testing/test-creation/country-of-registration/country-of-registration.html]: use falsy value

File: src/pages/testing/test-creation/test-create/test-create.ts [hunk 3]
```diff
@@ -22,6 +31,7 @@ export class TestCreatePage implements OnInit, OnDestroy {
   testTypesFieldsMetadata;
   testCompletionStatus;
   completedFields = {};
+  countriesArr = [];
```
Comment (danieliordache) [inline src/pages/testing/test-creation/test-create/test-create.ts]: use type

File: src/pages/testing/test-creation/country-of-registration/country-of-registration.ts (hunk 1 above)
Comment (vladddragomir) [reply src/pages/testing/test-creation/country-of-registration/country-of-registration.ts]: not needed

File: src/pages/testing/test-creation/test-create/test-create.ts (hunk 3 above)
Comment (vladddragomir) [reply src/pages/testing/test-creation/test-create/test-create.ts]: not needed

File: src/pages/testing/test-creation/country-of-registration/country-of-registration.html (hunk 2 above)
Comment (vladddragomir) [reply src/pages/testing/test-creation/country-of-registration/country-of-registration.html]: done

**Pull request merged**